src/tests/daos_perf.c

```diff
@@ -891,28 +1032,27 @@ static struct option ts_ops[] = {
 	{ NULL,		0,			NULL,	0   },
 };
 
-void show_result(double now, double then, int vsize, char *test_name)
+void show_result(double duration, uint64_t start, uint64_t end,
+		 int vsize, char *test_name)
 {
-	double		duration, agg_duration;
-	double		first_start;
-	double		last_end;
-	double		duration_max;
-	double		duration_min;
-	double		duration_sum;
-
-	duration = now - then;
+	double agg_duration;
+	double first_start;
+	double last_end;
+	double	duration_max;
+	double	duration_min;
+	double	duration_sum;
 
 	if (ts_ctx.tsc_mpi_size > 1) {
-		MPI_Reduce(&then, &first_start, 1, MPI_DOUBLE,
+		MPI_Reduce(&start, &first_start, 1, MPI_DOUBLE,
 			   MPI_MIN, 0, MPI_COMM_WORLD);
-		MPI_Reduce(&now, &last_end, 1, MPI_DOUBLE,
+		MPI_Reduce(&end, &last_end, 1, MPI_DOUBLE,
 			   MPI_MAX, 0, MPI_COMM_WORLD);
+		agg_duration = (last_end - first_start) / (1000 * 1000 * 1000);
 	} else {
-		first_start = then;
-		last_end = now;
+		agg_duration = duration /(1000 * 1000);
```

Comment:
(style)  need consistent spacing around '/' (ctx:WxV)

Reply:
need a space after the / to get rid of the warning